ExCSS.Tests/RuleLineFixture.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+
+namespace ExCSS.Tests
+{
+    [TestFixture]
+    public class RuleLineFixture
+    {
+        [Test]
+        public void Parser_Reads_One_Line()
+        {
+            var parser = new Parser();
+            var css = parser.Parse("html{color:#000000;}");
+            Assert.AreEqual(css.StyleRules[0].line, 1);
+        }
+        [Test]
+        public void Parser_Reads_Two_Lines_With_Line_Breaks()
+        {
+            var parser = new Parser();
```

Comment:
Could you add tests to check what happens if:
- You have an empty document
- You have a document with Windows line endings (\r\n)